isyfact-standards-doc/src/docs/antora/modules/isy-security/pages/konzept/inhalt.adoc

```diff
@@ -281,6 +281,46 @@ OpenID Connect Abfragen zur Identität eines Benutzers, OpenID-Token oder Custom
 === Authentifizierung & Autorisierung
 
+Gemäß den Vorgaben des IdentityAccessManagements (IAM), das in der  xref:isyfact-standards-doku:referenzarchitektur:fachlich.adoc#[IsyFact-Referenzarchitektur] zum Einsatz kommt, wird die Authentifizierung & Autorisierung über Multi-Realm bzw. Multi-Tenancy organisiert. Nach den vordefinierten xref:isyfact-standards-doku:referenzarchitektur:software-technisch.adoc#prinzipien-der-sicherheitsarchitektur[Prinzipien der Sicherheitsarchitektur] ergeben sich unterschiedliche Zugriffswege, wie das nachfolgende Bild zeigt:
+
+.Authentifizierungswege als Mandanten
+[id="image-multi-tenancy-multi-iam",reftext="{figure-caption} {counter:figures}"]
+image::konzept/security_multi-tenancy_with_multi_iams.dn.svg[align="center"]
+
+Die verschiedenen dargestellten Zugriffswege _Portal_, _Service-Gateway_, _Authentication-Gateway_ oder _interne Anwendungs-Authentifizierung_ (Batch/Task/Kontextwechsel) können als Tenants unterschieden werden. Dies wird - sofern gewünscht - durch den Sicherheitsbaustein _isy-security_ unterstützt.
+
+Die an den Zugriffwegen beteiligten Komponenten unterscheiden sich in ihrem Aufbau, ihrer Konfiguration und ihrer Funktionalität, so dass jede genau die speziellen Eigenheiten - siehe dazu Kapitel <<aufruf-weboberflaeche-portal>>, <<aufruf-service-gateway-ohne-authentication-gateway>>, <<aufruf-service-gateway-mit-authentication-gateway>> und <<ausfuehrung-batch-u-task>> bzw. <<aufruf-von-nachbarsystemen>> - am besten bedienen oder erfüllen kann.
+Die Umsetzung und sicherheitsrelevante Trennung dieser Zugriffswege werden als eigenständige Tenants betrachtet und können somit auch als mehrere Instanzen pro Zugriffsweg eingerichtet werden.
+Dadurch ist es möglich fachliche Unterschiede über den gleichen technischen Zugriffsweg bei der Authentifizierung zu machen.
+
+Wichtig ist zu beachten, dass alle Komponenten, die entlang eines Zugriffsweges auf den Tenant interagieren sollen, jeweils für sich die Tenant spezifische Konfiguration benötigen. Dies bedeutet zum Beispiel, dass eine Geschäftsanwendung, die ein  AccessToken von einem speziellen Tenant bei der Autorisierung validieren möchte, die Konfiguration des Tenant in der Anwendungseigenen Konfiguration vorliegen muss.
+
+Das nachfolgende Kapitel  erläutert diese Tenant spezifische Behandlung.
+
+[[ways-of-authentication-as-multi-tenant]]
+==== Authentifizierungswege als Tenants
+Die bisher aufgezeigten Authentifizierungswege werden in Bezug auf `isy-security` als jeweilig unterschiedliche Tenants behandelt bzw. konfiguriert.
+Aber auch pro Authentifizierungsweg ist es möglich unterschiedliche Tenants sprich mehrere Instanzen des gleichen Typs einzurichten.
+
+Um nun für jeden Tenant eine eindeutige Zuordnung bzw. eine Tenant spezifische Gruppierung der Properties herzustellen, muss der Property-Key im Pfad eine Kennung des Tenant beinhalten:
+
+`isy.security.oauth2.resourceserver.jwt.tenants.*<tenant-id>*`
```

Comment:
Die Konfigurationsparameter sind in den Nutzungsvorgaben bereits enthalten.

Reply:
In allgemeine Property-Formulierung geändert